src/candela/components/TrackerDash/ErrorBulletWidget.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'underscore';
+import d3 from 'd3';
+import nv from 'nvd3';
+import $ from 'jquery';
+
+import { computeColor } from './utility.js';
+import VisComponent from '../../VisComponent';
+
+class ErrorBulletWidget extends VisComponent {
```

Comment:
I'm ok with keeping Widget in the name for now. I agree we can change it later if needed.